skipper/data/skipper-entrypoint.sh

```diff
@@ -1,29 +1,32 @@
 #!/usr/bin/env bash
 
-SKIP_HOME_DIR_PARAM=""
-# if home directory already exists, useradd should not try to create it
-[ -d /home/${SKIPPER_USERNAME} ] && SKIP_HOME_DIR_PARAM="--no-create-home"
+if ! [ -z "${SKIPPER_DOCKER_GID}" ];then
```

Comment:
Podman is setting user byitself, so no need for any operation while we are using podman.
SKIPPER_DOCKER_GID is set only if we run with docker